packages/typespec-ts/test/modularIntegration/generated/lro_modular/generated/src/StandardClient.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  User,
+  OperationStatus,
+  ExportedUser,
+  beginCreateOrReplace,
+  beginDelete,
+  beginExport,
+  CreateOrReplaceOptions,
+  DeleteOptions,
+  ExportOptions,
+  createStandard,
+  StandardContext,
+  StandardClientOptions,
+} from "./api/index.js";
+import { SimplePollerLike, OperationState } from "@azure/core-lro";
+
+export { StandardClientOptions } from "./api/StandardContext.js";
+
+export class StandardClient {
+  private _client: StandardContext;
+
+  /** Illustrates bodies templated with Azure Core with long-running operation */
+  constructor(options: StandardClientOptions = {}) {
+    this._client = createStandard(options);
+  }
+
+  /** Creates or replaces a User */
+  beginCreateOrReplace(
+    role: string,
+    name: string,
+    options: CreateOrReplaceOptions = { requestOptions: {} }
+  ): Promise<SimplePollerLike<OperationState<User>, User>> {
+    return beginCreateOrReplace(this._client, role, name, options);
+  }
```

Comment:
missing a beginCreateOrReplaceAndWait for short lro.

Reply:
https://github.com/Azure/autorest.typescript/pull/1862#discussion_r1204556819